memor/__init__.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 """Memor modules."""
 from .params import MEMOR_VERSION, RenderFormat
+from .token_estimators import TokenEstimator
```

Comment:
@sadrasabouri 
Isn't it better to use `tokens_estimators.py` and `TokensEstimator` instead of `token_estimators.py` and `TokenEstimator`? It's not a big issue; I just mentioned it for consistency.

Reply:
My suggestion:

It's better to consistently use either `tokens` or `token` throughout the project.